_erfa assertion error with 3.2rc1 using numpy 1.16

>     def test_output_dim_3_signature():
>         if NUMPY_LT_1_16:
>             assert erfa_ufunc.c2i00a.signature == "(),()->(d3, d3)"
>         else:
>            assert erfa_ufunc.c2i00a.signature == "(),()->(3, 3)"
>  E           AssertionError: assert '(),()->(d3, d3)' == '(),()->(3, 3)'
>  E             - (),()->(d3, d3)
>  E             ?         -   -
>  E             + (),()->(3, 3)
> 
>  /miniconda3/envs/imexam/lib/python3.7/site-packages/astropy/_erfa/tests/test_erfa.py:14: >  AssertionError


Platform: Darwin-17.7.0-x86_64-i386-64bit
Full Python Version: 
3.7.2 (default, Dec 29 2018, 00:00:04) 
[Clang 4.0.1 (tags/RELEASE_401/final)]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Numpy: 1.16.2
Scipy: 1.2.1
Matplotlib: 3.0.3
h5py: 2.9.0
Pandas: 0.24.2
Asdf: 2.4.0.dev1734
Cython: 0.29.6
Scikit-image: 0.14.2
astropy_helpers: 3.2rc1
Using Astropy options: remote_data: any.

I don't remember do that. Just to be clean, I just created a new environment with py3.7, installed numpy, installed pytest-astropy, pip install astropy==3.2rc1, and reran the tests. This is what I get:

`1 failed, 11259 passed, 773 skipped, 69 xfailed, 2445 warnings in 200.65 seconds`

================================================================= FAILURES ==================================================================
________________________________________________________ test_output_dim_3_signature ________________________________________________________

    def test_output_dim_3_signature():
        if NUMPY_LT_1_16:
            assert erfa_ufunc.c2i00a.signature == "(),()->(d3, d3)"
        else:
>           assert erfa_ufunc.c2i00a.signature == "(),()->(3, 3)"
E           AssertionError: assert '(),()->(d3, d3)' == '(),()->(3, 3)'
E             - (),()->(d3, d3)
E             ?         -   -
E             + (),()->(3, 3)

/miniconda3/envs/test-astropy/lib/python3.7/site-packages/astropy/_erfa/tests/test_erfa.py:14: AssertionError
============================================================= warnings summary ==============================================================
/miniconda3/envs/test-astropy/lib/python3.7/importlib/_bootstrap.py:219
  /miniconda3/envs/test-astropy/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
    return f(*args, **kwds)


with

platform darwin -- Python 3.7.3, pytest-4.4.2, py-1.8.0, pluggy-0.11.0

Running tests with Astropy version 3.2rc1.
Running tests in /miniconda3/envs/test-astropy/lib/python3.7/site-packages/astropy.

Date: 2019-05-15T11:32:57

Platform: Darwin-17.7.0-x86_64-i386-64bit

Executable: /miniconda3/envs/test-astropy/bin/python

Full Python Version: 
3.7.3 (default, Mar 27 2019, 16:54:48) 
[Clang 4.0.1 (tags/RELEASE_401/final)]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Numpy: 1.16.3
Scipy: not available
Matplotlib: not available
h5py: not available
Pandas: not available
Cython: not available
Scikit-image: not available
astropy_helpers: 3.2rc1
Using Astropy options: remote_data: any.

rootdir: /miniconda3/envs/test-astropy/lib/python3.7/site-packages/astropy
plugins: remotedata-0.3.1, openfiles-0.3.2, doctestplus-0.3.0, arraydiff-0.3

--- a/astropy/_erfa/tests/test_erfa.py
+++ b/astropy/_erfa/tests/test_erfa.py
@@ -2,16 +2,8 @@
 
 import numpy as np
 
-from astropy._erfa import core as erfa, ufunc as erfa_ufunc
+from astropy._erfa import core as erfa
 from astropy.tests.helper import catch_warnings
-from astropy.utils.compat import NUMPY_LT_1_16
-
-
-def test_output_dim_3_signature():
-    if NUMPY_LT_1_16:
-        assert erfa_ufunc.c2i00a.signature == "(),()->(d3, d3)"
-    else:
-        assert erfa_ufunc.c2i00a.signature == "(),()->(3, 3)"
 
 
 def test_erfa_wrapper():

